pop-api/integration-tests/Cargo.toml

```
@@ -1,36 +1,39 @@
 [package]
 build = "build.rs"
 edition = "2021"
-name = "integration-tests"
+name = "pop-api-integration-tests"
 version = "0.1.0"
 
 [build-dependencies]
-contract-build = "4.1.1"
+contract-build.workspace = true
 
 [dev-dependencies]
-env_logger = "0.11.2"
-frame-support = { version = "36.0.0", default-features = false }
-frame-system = { version = "36.1.0", default-features = false }
-log = "0.4.22"
-pallet-assets = { version = "37.0.0", default-features = false }
-pallet-balances = { version = "37.0.0", default-features = false }
-pallet-contracts = { version = "35.0.0", default-features = false }
+env_logger.workspace = true
+log.workspace = true
+
+codec = { workspace = true, default-features = false, features = [
```

As for the rest of the dependencies, because in the root Cargo.toml file they are declared already with default features false, you don't have to do it again here.
```suggestion
codec = { workspace = true, features = [ "derive" ] }
```